src/bin/ff-compute-accuracy.rs

```diff
@@ -127,18 +129,25 @@ impl<'a> Eval<'a> {
 
         // If the model is not able to provide a query result, it is counted
         // as an error.
-        let is_correct = self
+        let (is_correct, cos) = self
             .embeddings
             .analogy(&instance.query.0, &instance.query.1, &instance.query.2, 1)
-            .map(|r| r.first().unwrap().word == instance.answer)
-            .unwrap_or(false);
+            .map(|r| {
+                let result = r.first().unwrap();
+                (
+                    result.word == instance.answer,
+                    result.similarity.into_inner(),
+                )
+            })
+            .unwrap_or((false, -1.));
```

Comment:
I am wondering if an orthogonal vector is fairer. In the case of a completely nonsensical vector space (e.g. random vectors), the average cosine similarity would be 0.

Reply:
Sure, the -1 was based on the intuition to use the value with maximum cosine distance from the target.  I pushed 0 default.